Updated setup-tools topic per issue. (#338)
Concordium/concordium.github.io: source/shared/smart-contracts/guides/setup-tools.rst

   Add links for testing and schemas.

Cargo Concordium is distributed as part of the :ref:`Concordium software<downloads>` package. Rename the downloaded file to ``cargo-concordium`` for MacOS/Linux, and ``cargo-concordium.exe`` for Windows.

The tool should be placed in your PATH. This can be achieved by either:

* Moving the executable to a folder that is already on your PATH, for example ``%HOMEPATH%\.cargo\bin\`` for Windows, and ``$HOME/.cargo/bin`` for MacOS/Linux.
* Or, by adding its current location to your PATH environment.

> If you are running MacOS, you need to make the tool executable by running ``chmod +x path/to/cargo-concordium_1.0.0-0`` in a terminal. Make sure to provide the correct path to the downloaded tool. You also need to grant it permission to run in your  `Security & Privacy settings <https://support.apple.com/en-gb/guide/mac-help/mh40616/mac>`_ .

Reviewer: ```suggestion
If you are running MacOS, you need to make the tool executable by running ``chmod +x path/to/cargo-concordium` in a terminal. Make sure to provide the correct path to the downloaded tool. You also need to grant it permission to run in your  `Security & Privacy settings <https://support.apple.com/en-gb/guide/mac-help/mh40616/mac>`_ .
```

Would make sense to remove the version information here as well, if the user should have already renamed the file.

   Add links for testing and schemas.

Cargo Concordium is distributed as part of the :ref:`Concordium software<downloads>` package.
The tool should be placed in your PATH. This can be achieved by either:

> 

Reviewer: ```suggestion

* Rename the downloaded file to ``cargo-concordium`` for MacOS/Linux, and ``cargo-concordium.exe`` for Windows.
```

The original issue from our support forum also mentions the need to rename the file. This also ensures, that the file correctly replaces already existing versions when placed in the same folder.

> * Moving the executable to a folder that is already on your PATH, for example ``C:\Users\<your-username>\.cargo\bin\`` (replacing <your-username> with your actual username).

Reviewer: ```suggestion
* Moving the executable to a folder that is already on your PATH, for example ``%HOMEPATH%\.cargo\bin\`` for Windows, and ``$HOME/.cargo/bin`` for MacOS/Linux
```

The folder path is only valid for windows. I guess it might be fine as it is an example as to where it could be placed. Also I think it might be nice to use the home directory variables instead of username replacement.

* Or, by adding its current location to your PATH environment.

> Once cargo-concordium.exe is on your path, you can invoke it with ``cargo concordium`` in the terminal.
> 
> If you are running MacOS, you need to make the tool executable by running ``chmod +x path/to/cargo-concordium_1.0.0-0`` in a terminal. Make sure to provide the correct path to the downloaded tool. You also need to grant it permission to run in your  `Security & Privacy settings <https://support.apple.com/en-gb/guide/mac-help/mh40616/mac>`_ .

Reviewer: ```suggestion
If you are running MacOS, you need to make the tool executable by running ``chmod +x path/to/cargo-concordium`` in a terminal. Make sure to provide the correct path to the downloaded tool. You also need to grant it permission to run in your  `Security & Privacy settings <https://support.apple.com/en-gb/guide/mac-help/mh40616/mac>`_ .

Once the cargo-concordium executable is on your path, you can invoke it with ``cargo concordium`` in the terminal.
```

I think it makes more sense, if the MacOS note on making it executable comes before the "now you can execute in the terminal by running" statement.
